internal/app/machined/pkg/controllers/kubespan/peer_spec.go

```diff
@@ -127,9 +127,22 @@ func (ctrl *PeerSpecController) Run(ctx context.Context, r controller.Runtime, l
 						continue
 					}
 
+					ipSetPrefixes := ipSet.Prefixes()
```

Comment:
should we extract a function which stringifies `IPSet` and use it with logging?

as I don't think we need to build this list if there're no overlaps (and we can build it only when logging)

Reply:
Sounds good.

Do we want to log that by default? Or only in debug builds?

Reply:
I think by default is good!